### ISSUE TITLE
Deprecated tslint. Any plan to migrate it?

### ISSUE THREAD

**Issue by brucebrit** [maximegris/angular-electron#400]
Hi,

i read that "Palantir, the backers behind TSLint announced earlier this year that they would be deprecating TSLint in favor of supporting typescript-eslint in order to benefit the community. 
The TypeScript Team themselves also announced their plans to move the TypeScript codebase from TSLint to typescript-eslint, and they have been big supporters of this project. "

In that case, any plan to update the package.json to use the typescript-eslint instead of tslint ?

**Comment by whyboris** [comment]
Looks like the [official _GitHub_ page](https://github.com/palantir/tslint#tslint) states this now:

> TSLint will be deprecated some time in 2019. See this issue for more details: [Roadmap: TSLint → ESLint.](https://github.com/palantir/tslint/issues/4534)

The link points to a [blog post](https://medium.com/palantir/tslint-in-2019-1a144c2317a9) describing the scenario.

TL;DR: **migrate to [ESLint](https://github.com/eslint/eslint)** 🤷‍♂

**Comment by cheplv** [comment]
@whyboris think that this needs to be done, will find some 4-6 hours to make this change in 2-3 weeks

**Comment by cheplv** [comment]
@whyboris initial work is done - there is some warnings now on linting.
Check PR: https://github.com/maximegris/angular-electron/pull/410

**Comment by maximegris** [comment]
Thanks guys for the time spent on this subjet 👍 
UnfortunatIy I had no time to work on it.
So thanks a lot @cheplv for the PR (Merged) !! 🎉

**Comment by cheplv** [comment]
@maximegris this is the power of open source projects ))

**Comment by maximegris** [comment]
I totally agree :D

**Comment by cheplv** [comment]
@brucebrit think that this ticket can be closed due migration changes in master

**Comment by brucebrit** [comment]
Thanks everyone
